### ISSUE TITLE
Support installing drivers from file

### ISSUE THREAD

**Issue by kafkasl** [bblfsh/bblfshd#181]
Support installing drivers from a file. Useful for systems (like many supercomputers) without direct access to internet. Could be something like:

`bblfshctl driver install Python python-driver/ --ctl-address ~/bblfsh/bblfshctl.sock`

**Comment by juanjux** [comment]
I agree this would be a really helpful adition to bblfshd, will try to priorize it since the "copy directory" kludge doesn't always seem to work. 

Ideally, this would install from an exported docker filesystem image since the source code driver directory would need a build and other operations that could be problematic in restricted computers.
